src/aml_service/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const yotiRequest = require('../yoti_request');
+const AmlResultClass = require('../yoti_request/aml.result').AmlResult;
+const Payload = require('../yoti_request/payload').Payload;
+
+const AmlResultObj = function(rawResult){
+  this.onPepList = rawResult['on_pep_list'];
+  this.onFraudList = rawResult['on_fraud_list'];
+  this.onWatchList = rawResult['on_watch_list'];
+}
+
+AmlResultObj.prototype = {
+  /**
+   * Check if user is a politically exposed person.
+   *
+   * @returns {exports.AmlResult.isOnPepList}
+   */
+  isOnPepList() {
+    return this.onPepList;
+  },
+
+  /**
+   * Check if user is on a fraud list.
+   *
+   * @returns {exports.AmlResult.isOnFraudList}
+   */
+  isOnFraudList () {
+    return this.onFraudList;
+  },
+
+  /**
+   * Check if user is on a watch list.
+   *
+   * @returns {exports.AmlResult.isOnWatchList}
+   */
+  isOnWatchList () {
+    return this.onWatchList;
+  },
+
+  /**
+   * Get result data.
+   *
+   * @returns {{}}
+   */
+  getData () {
+    return {
+      'on_pep_list': this.isOnPepList(),
+      'on_fraud_list': this.isOnFraudList(),
+      'on_watch_list': this.isOnWatchList()
+    };
+  },
+
+  toString () {
+    return JSON.stringify(this.getData());
+  }
+}
+
+exports.performAmlCheck = (amlProfile, pem, appId) => {
```

Comment:
This might need a rewrite, but `performAmlCheck` should not require `pem` and `appId`.

Reply:
We need appId don't we? 
I was working from `/aml-check?appId={app_id}&timestamp={timestamp}&nonce={nonce}` from https://lampkicking.atlassian.net/wiki/spaces/IPASS/pages/139068116/Connect+API+AML+Endpoints

Reply:
Yes, but the request should already know what the value of appId is. There's no point sending it from all the services that make a request. The use of appId is debatable, but pem definitely does not belong there.

Reply:
This is following the same approach used in `ActivityDetails.getReceipt`, I don't see anything wrong with it. And I don't see why loading `pem` file in some kind of global configuration will make it any better.

Reply:
The request itself can receive pem and appId arguments. These values are global, they might be used outside of this request, that's why it's better using them in a configuration object. You can see that an AML check does not require a pem file, but the request will ask for one. This is not the correct behaviour.